prime-router/src/test/kotlin/fhirengine/azure/FHIRRouterIntegrationTests.kt

```diff
@@ -628,15 +626,19 @@ class FHIRRouterIntegrationTests : Logging {
         checkActionTable(listOf(TaskAction.convert, TaskAction.receive))
 
         // execute
-        var receivers = createReceivers(listOf(ReceiverSetupData("x", conditionFilter = observationFilter)))
+        var receiverSetupData = listOf(
+            ReceiverSetupData("x", conditionFilter = observationFilter),
+            ReceiverSetupData("y"),
+        )
+        var receivers = createReceivers(receiverSetupData)
         val org = createOrganizationWithReceivers(receivers)
         val fhirRouter = createFHIRRouter(org)
         fhirFunctions.doRoute(queueMessage, 1, fhirRouter)
 
         // check results
         ReportStreamTestDatabaseContainer.testDatabaseAccess.transact { txn ->
             // did the report get pushed to blob store correctly and intact?
-            val routedReports = verifyLineageAndFetchCreatedReportFiles(convertReport, receiveReport, txn, 1)
+            val routedReports = verifyLineageAndFetchCreatedReportFiles(convertReport, receiveReport, txn, 2)
```

Comment:
For me locally, the test actually fails here and none of the assertions against the bundle are getting run

Reply:
✅